examples/v1/main.go

```diff
@@ -52,7 +52,12 @@ func main() {
 	}
 	defer fh.Close()
 
-	err = cl.UpdateBMCFirmware(ctx, fh)
+  fi, err := fh.Stat()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = cl.UpdateBMCFirmware(ctx, fh, fi.Size())
```

Comment:
These are a bit more related as they both pertain to the `examples/...` code, but as with bd9fd26ceca67d363e3fc23b579a972d093a7af0 in an ideal world I think these changes would be two separate commits.

Reply:
...though actually, looking at the change here it seems like this code wouldn't have compiled prior to this change?  In which case maybe this should have just been a part of 2055fe1d1d6de1bd3c47080ecd1197f2abc18d27...

Reply:
fixed, now is a part of 808952d2

Reply:
As is it still looks like there's compilation breakage here -- this commit adds the third argument to the `UpdateBMCFirmware()` call, but that parameter isn't added to the function until 68546ed177468ab67569a65ad9a101e173f78c5f?

Reply:
edited 808952d as e0b562ef6 and moved examples into fba84da